hieradata/clients/census.yaml

```diff
@@ -0,0 +1,9 @@
+---
+lvm::volume_groups:
+  data:
+    physical_volumes:
+      - /dev/xvdb
```

Comment:
This might not be the device name of the EBS volume as mentioned in [INFRA-676](https://issues.jenkins-ci.org/browse/INFRA-676)

So I'll make sure that the machine is bootstrapped prior to merging this.